src/hooks/useLocalCurrencyPrice.ts

```diff
@@ -0,0 +1,27 @@
+import { Currency } from 'graphql/data/__generated__/types-and-hooks'
+import { useLocalCurrencyConversionRate } from 'graphql/data/ConversionRate'
+
+import { useActiveLocalCurrency } from './useActiveLocalCurrency'
+import { useUSDPrice } from './useUSDPrice'
+
+type useUSDPriceParameters = Parameters<typeof useUSDPrice>
+
+export function useLocalCurrencyPrice(...useUSDPriceParameters: useUSDPriceParameters) {
+  const activeLocalCurrency = useActiveLocalCurrency()
+  const activeLocalCurrencyIsUSD = activeLocalCurrency === Currency.Usd
+
+  const { data: usdPrice, isLoading: isUSDPriceLoading } = useUSDPrice(...useUSDPriceParameters)
+  const { data: localCurrencyConversionRate, isLoading: isLocalCurrencyConversionRateLoading } =
+    useLocalCurrencyConversionRate(activeLocalCurrency, activeLocalCurrencyIsUSD)
```

Comment:
should we skip this fetch if `activeLocalCurrencyIsUSD` is true?

Reply:
yeah thats exactly what happens